scripts/assets/Swift File For Package.xctemplate/___FILEBASENAME___.swift

```diff
@@ -1,3 +1,3 @@
-//___FILEHEADER___
+// ___FILEHEADER___
```

Comment:
Wait, does this one have side effects

Reply:
To be honest, I'm not sure this is even being used - let me check though.

Reply:
Okay I undid this change just to be safe